### PR TITLE
Improve layout and design of sidebar

### PR DESCRIPTION
**Pull request by mattiastofte**
First change of the planned improvements of the /pages/ route layout. 

This PR includes:
- Improved alignment of list items
- New icon for dropdown buttons
- Flip animation/transition for icon
- Cleaner design with small tint of background instead of right border/divider
- Links are gray while the "abakule" circle is kept red
- Mobile look is vastly improved (although not perfect)

Light mode:
image

Dark mode:
image

### CODE REVIEW

**Comment by Arashfa0301** [comment]
Could you include an illustration of the mobile view in this PR ?

File: app/routes/pages/components/Sidebar.js [hunk 1]
```diff
@@ -51,9 +51,10 @@ class Sidebar extends Component<Props, State> {
                 <Icon name="close" size={50} />
               </button>
               <h3 className={styles.sidebarHeader}>Om Abakus</h3>
+              {/*
               <h4 className={styles.sidebarSubtitle}>
                 {categorySelected ? categorySelected : 'Generelt'}
-              </h4>
+              </h4>*/}
```
Comment (Arashfa0301) [inline app/routes/pages/components/Sidebar.js:57]: Do you need this comment here ?

Comment (ivarnakken) [reply app/routes/pages/components/Sidebar.js:57]: I'd say you can just remove it entirely.

**Comment by LudvigHz** [comment]
Nice! :100: 
Some of my thoughts:
- This new design looks a lot _cleaner_ (especially on dark mode).
- However, IMO it might be a bit _too clean_. In that I miss some contrast and indication that the different menu items are clickable. Could f.ex. having some background for either theall the different items, or just the selected one work?
Here is one example with a border between each item, and some different color on hover:
image

- It still looks kinda off to me that the red bullet point is inset in the list item :thinking:. I wonder how it would look if it was just before the text without "shifting" it to the right. I.e. like this:
  image

**Comment by LudvigHz** [comment]
Another Idea I had is to also change the bullet point with a border, if you go for the "block" click target as above (i.e. each element is more a button than just a link).
Something like this:
image

**Comment by mattiastofte** [comment]
Great feedback! I agree that the design could be more readable so I'll keep experimenting with different approaches to achieve this. Although I think it's kinda cool to use a red circle for indication, since it emphasises the abakule:)

**Comment by eikhr** [comment]
I really like the newest design, however I think the border between items would look better if they were lighter than the background in dark mode.

<table>
<tr>
	<td>Like this</td>
	<td>Instead of this</td>
<tr>
	<td>

Screenshot 2022-10-24 at 14 54 22
</td>
	<td>

Screenshot 2022-10-24 at 14 54 31
</td>
</table>

**Comment by ivarnakken** [comment]
@mattiastofte Is this turning stale? I feel like we had something very nice. If you're bored of working on it, we can fix up the small stuff, get it merged in, and then iterate on it later. I think anything is better than the current design.

**Comment by mattiastofte** [comment]
It's kinda stale since the branch is quite far behind the main branch so I'd have to rebase and I'm not super confident with that. I also would like to improve the design futher as I've gotten some ideas and I've done something similar to the sidenav og kateter.no. I'll look at it tonight!

**Comment by mattiastofte** [comment]
> @mattiastofte Is this turning stale? I feel like we had something very nice. If you're bored of working on it, we can fix up the small stuff, get it merged in, and then iterate on it later. I think anything is better than the current design.
> 
> 
> 
> BTW: I think it would look good to use the `var(--additive-background)` as a hover background, and no border.

It's kinda stale since the branch is quite far behind the main branch so I'd have to rebase and I'm not super confident with that. I also would like to improve the design futher as I've gotten some ideas and I've done something similar to the sidenav og kateter.no. I'll look at it tonight!

**Comment by ivarnakken** [comment]
>I also would like to improve the design futher as I've gotten some new ideas and I've done something similar to the sidenav on kateter.no. I'll look at it tonight!

Awesome! 💯 Excited to see the result 😄 Let me know if you need any help rebasing.

**Comment by ivarnakken** [comment]
Closed in favor of #3553

**Pull request closed without merging**